apiutils/utils.go

```diff
@@ -291,6 +292,44 @@ func GetGoogleOAuthClientID(ctx context.Context, api string, tlsConfig *tls.Conf
 	return ioutil.ReadAll(resp.Body)
 }
 
+// GetTime returns the current time from the api server.
```

Comment:
We need unittests @philipatl 🙏

Reply:
@t00f Are there unit tests for similar functions here?

Reply:
Nope. Since you are the first one to touch that in years, you won the right to write them :)